pkg/solana/types_test.go

```diff
@@ -37,46 +37,41 @@ func TestState_Decode(t *testing.T) {
 
 var mockRawState = []byte{
```

Comment:
Consider declaring the config digest for this mocked state so we can reference it in tests from one place:

```go
var mockState :=   struct {
        Raw                          []byte
        ConfigDigestHex    string
    }{
        Raw:                          []byte{...},
        ConfigDigestHex:    "00039ad1eaa13649831b4cc02f5437bf556910037c6833f87249edbeceab2828",
    }
```

Or even declare a type if we'll have more than one test example.